source/common/common/interval_value.h

```diff
@@ -25,10 +25,11 @@ template <typename T, typename Interval> class ClosedIntervalValue {
   // Returns a value that is as far from max as the original value is from min.
   // This guarantees that max().invert() == min() and min().invert() == max().
   ClosedIntervalValue invert() const {
-    return ClosedIntervalValue(value_ == Interval::max_value ? Interval::min_value
-                               : value_ == Interval::min_value
-                                   ? Interval::max_value
-                                   : Interval::max_value - (value_ - Interval::min_value));
+    return ClosedIntervalValue(value_ == Interval::max_value
+                                   ? Interval::min_value
+                                   : value_ == Interval::min_value
+                                         ? Interval::max_value
+                                         : Interval::max_value - (value_ - Interval::min_value));
```

Comment:
@stevenzzzz format is failing, and it seems this is a gratuitous change, so maybe take a look into that.